BDArmory/Radar/VesselRadarData.cs

```diff
@@ -926,7 +926,11 @@ private void WindowRadar(int windowID)
             //==============================
             GUI.BeginGroup(RadarDisplayRect);
 
-            if (availableRadars.Count + availableIRSTs.Count == 0) return;
+            if (availableRadars.Count + availableIRSTs.Count == 0)
```

Comment:
Couldn't this just go before the `GUI.BeginGroup(RadarDisplayRect);`? It seems unnecessary to begin a group then immediately end it.

Reply:
yes